pkg/operator/operator.go

```diff
@@ -88,6 +105,16 @@ func (o *Operator) Run() error {
 		<-time.After(initRetryDelay)
 	}
 
+	// Run volume provisioner
+	// The controller needs to know what the server version is because out-of-tree
```

Comment:
Yes i like that. I rather the user (when using K8s 1.5) encounter the failure early and not later.